server/publicapi/packages/service.py

```diff
@@ -9,13 +9,13 @@
 # at https://www.sourcefabric.org/superdesk/license
 
 import logging
-from superdesk.services import BaseService
+from publicapi.items.service import ItemsService
 
 
 logger = logging.getLogger(__name__)
 
 
-class PackagesService(BaseService):
+class PackagesService(ItemsService):
```

Comment:
"Hm... I don't like it that much but OK. My main concern is that at some point somebody will forget to decouple the relation if the functionality diverges (it's always like that :smile: )."

That's called refactoring :) It's about detecting when code structure does not match the functionality :)